apps/extension/src/App/Accounts/AccountListing.tsx

```diff
@@ -6,43 +7,59 @@ import {
   Details,
   Alias,
   DerivationPath,
-  CopyToClipboard,
+  Button,
 } from "./AccountListing.components";
 
-import { DerivedAccount } from "background/keyring";
+import { DerivedAccount, KeyStoreType } from "background/keyring";
+import { TopLevelRoute } from "App/types";
 
 type Props = {
+  // The parent Bip44 "account"
+  parent?: number;
+  // The child account
   account: DerivedAccount;
+  parentAlias: string;
```

Comment:
very nit: `parentAlias` gets a value that is based on `alias` which is optional in `DerivedAccount`. It would be cleaner to treat it as such until it is being utilised in this file and remove the logic to turn a possible undefined to `""` in the parent. Because now in the parent we do not know why the logic is there without reading this file.